test/multiverse/lib/multiverse/suite.rb

```diff
@@ -12,7 +12,6 @@
 require 'base64'
 require 'fileutils'
 require 'digest'
-require_relative '../multiverse'
```

Comment:
`../multiverse` itself requires this class. Removing this line fixes a circular dependency.